packages/compartment-mapper/src/link.js

```diff
@@ -95,12 +95,7 @@ const makeExtensionParser = (
     ) {
       language = languageForModuleSpecifier[specifier];
     } else {
-      if (!has(languageForExtension, extension)) {
-        throw new Error(
-          `Cannot parse module ${specifier} at ${location}, no parser configured for extension ${extension}`,
-        );
-      }
-      language = languageForExtension[extension];
+      language = languageForExtension[extension] || `extension:${extension}`;
     }
 
     if (has(transforms, language)) {
```

Comment:
this could be a `while` loop instead

Reply:
>   107:38  error  Unexpected `await` inside a loop  no-await-in-loop

Reply:
hmm actually this would break for things that dont change the language type but expect to run only once and not infinite times... :skull: